modules/ROOT/pages/errors/gql-errors/08N13.adoc

```diff
@@ -1,4 +1,12 @@
 = 08N13
 
 == Status description
-error: connection exception - database not up to requested bookmark. The database `{ $db }` is not up to the requested bookmark `{ $transactionId1 }`. The latest transaction ID is `{ $transactionId2 }`.
+error: connection exception - database not up to requested bookmark. The database `{ <<db>> }` is not up to the requested bookmark `{ <<transactionId>>1 }`. The latest transaction ID is `{ $transactionId2 }`.
```

Comment:
```suggestion
error: connection exception - database not up to requested bookmark. The database `{ <<db>> }` is not up to the requested bookmark `{ <<transactionId>>1 }`. The latest transaction ID is `{ <<transactionId>>2 }`.
```

If I remember correctly, what we said last week is that if we have several of the same parameter, we would make a hoverover for each like this

Reply:
I think it won't benefit the user to see the same description twice. Let's talk about it again.

Reply:
Sure, I don't have a strong opinion, just repeating what Gowri said